recipes/r-gsw/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.0-3" %}
+
+package:
+  name: r-gsw
```

Comment:
If you use the latest version of conda-build, you should get an updated `conda skeleton cran` that provides recipes in conda-forge style. It will e.g. also define a variable for the package name.